### DISCUSSION TITLE
How do I get followers and following fast?

### DISCUSSION THREAD

**Discussion by aslios** [sbertix/Swiftagram#156] [Q&A] [answered]
Hi Sbertix
i saw some applications about instagram tacking.For example app names are SARMAN , Repost+ etc. They are getting followers and following very fastly.I am sure they are not using Instagram Official API but How can they get these datas fastly? Can we get followers and followings fastly like these apps via your api or how?

**Comment by sbertix** [comment, marked as answer]
Hey @aslios,

Fast followers/following fetchers tend to rely on scrapers for the Instagram mobile website. 
The actual implementation is pretty straightforward, but it's beyond the scopes of this repo. 

I might decide to publish my own code with it at one point, but I'm not planning on releasing it any time soon.

**Reply by aslios** [reply]
Can you give me some guidance on how to do this? I've been working on this for a long time, but I haven't had any correct results:(

**Reply by sbertix** [reply]
I'm sorry. 

Like I said, I'm not really comfortable in releasing this at this point time, and there's not much more I can tell you, without sharing any code, as I've already pointed you in the right direction, telling you about the scraper. 

Again, I might open source this eventually, and I'll make sure to let you know, would you still be interested in it. 

Sorry @aslios

**Reply by aslios** [reply]
of course please let me know :(
